provision/pull-images.sh

```diff
@@ -49,7 +49,7 @@ if [ -z "${NAME_PREFIX}" ]; then
         docker.io/prom/statsd-exporter:v0.6.0 \
         docker.io/tgraf/netperf:v1.0 \
         docker.io/wurstmeister/kafka:1.1.0 \
-        docker.io/nebril/python-binary-memcached \
+        docker.io/cilium/python-bmemcached:v0.0.1 \
```

Comment:
Thanks!

One question though: where can we find Dockerfile for this image?

Reply:
+1, having this as a repo under the cilium organization on GitHub would be great as well.

Reply:
In https://hub.docker.com/r/cilium/python-bmemcached

```
Source Repository
Github
cilium/python-bmemcached
```